index.js

```diff
@@ -35,6 +43,16 @@ function gulpAmdCheck(options) {
         file.contents = new Buffer(output.optimizedContent);
       }
 
+      if (options.errorOnUnusedDependencies && output.results.length) {
+        var unusedDependencies = output.results.map(function (res) { return res.unusedDependencies });
+
+        unusedDependencies = [].concat.apply([], unusedDependencies); // Flatten array
+
+        if (unusedDependencies.length) {
+          missingDependenciesDetected.push(formatError(file, [].concat.apply(unusedDependencies)));

```

Comment:
`[].concat.apply(unusedDependencies)` can be replaced by `unusedDependencies`. Right?